.release-please-manifest.json

```diff
@@ -1,7 +1,7 @@
 {
-  "crates/vee": "0.1.0",
+  "crates/vee": "0.1.1",
   "crates/decoder": "0.1.0",
-  "crates/header-accumulator": "0.1.0",
+  "crates/header-accumulator": "0.2.0",
   "crates/firehose-protos": "0.1.0",
   "crates/firehose-protos-examples": "0.1.0"
```

Comment:
We don't need to manipulate this while we roll back the issue with the reth-trie/alloy-trie dependency issue. Let's leave these alone.